### ISSUE TITLE
I need know the swf is play to the end.

### ISSUE THREAD

**Issue by tidus2020** [ruffle-rs/ruffle#9032]
My swf is animate only , and now it is loop and i don't know when is finish.

**Comment by n0samu** [comment]
I agree that it would be a good idea for Ruffle to fire JavaScript events for things like this (the SWF finishing playback). For now though, you might want to edit the SWF file itself to force it to stop looping. (In Flash you could use an embed attribute to enable/disable looping, but Ruffle does not support this yet - see #4258)
